torchdet3d/__init__.py

```diff
@@ -2,11 +2,11 @@
 import sys
 import os
 
-module_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../3rdparty/Objectron'))
```

Comment:
Why do you need this change?

Reply:
because it runs with scripts in cache copied by setup.py and `__file__` gives the wrong path to __init__.py and it hasn't worked like that no matter how hard I try. If change it to `__init__.py` it works as expected.